tests/test_averell.py

```diff
@@ -2,6 +2,7 @@
 
 from averell.cli import download
 from averell.cli import export
+from averell.cli import list
```

Comment:
Overwriting the native `list()` function might cause unexpected side effects. Can we change the name of the function while still preserving the command?

Reply:
Done